include/swift/AST/DiagnosticsSema.def

```diff
@@ -5127,6 +5127,10 @@ NOTE(function_builder_infer_add_return, none,
 NOTE(function_builder_infer_pick_specific, none,
      "apply function builder %0 (inferred from %select{protocol|dynamic replacement of}1 %2)",
      (Type, unsigned, DeclName))
+WARNING(function_builder_missing_limited_availability, none,
+        "function builder %0 does not implement `buildLimitedAvailability`; "
```

Comment:
```suggestion
        "function builder %0 does not implement 'buildLimitedAvailability'; "
```

It would also be great to put this in the YAML diagnostic localization file :)